vhdbuilder/packer/pre-install-dependencies.sh

```diff
@@ -48,6 +48,18 @@ stop_watch $capture_time "Set Permissions if Mariner" false
 start_watch
 
 copyPackerFiles
+
+# Update rsyslog configuration
+RSYSLOG_CONFIG_FILEPATH="/etc/rsyslog.d/60-CIS.conf"
+if [[ $OS == $MARINER_OS_NAME ]]; then
+    echo -e "\nnews.none                          -/var/log/messages" >> ${RSYSLOG_CONFIG_FILEPATH}
```

Comment:
do we actually need all this blank space?

Reply:
This blank space is added to ensure that this line aligns with the spacing of other lines in 60-CIS.conf file.